.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug
+about: Bug Issue Template
+title: '[Bug]'
+labels: development, bug
+assignees: ''
+---
+
+**Description-**
+
+**Type of Bug-**
+
+- [ ] Accessibility
+- [ ] Functional
+- [ ] Usability
+- [ ] API
+- [ ] Regression
+
+**Role Affected-** (If not stated, ALL)
+
+**Browser-** (if it only happens in a single browser)
```

Comment:
We explicitly only support FF, Chrome and IE, maybe put this below repro steps and include checkboxes for "Duplicated in:" [] FF, [] Chrome, [] IE